docs/tutorial/kymotracking.rst

```diff
@@ -387,3 +387,100 @@ By default, this will use the optimal number of lags (which in this case seems t
+
+The normalization constant :math:`N` takes into account the minimum and maximum possible observation times of
+a track. In order to estimate these bounds, the method uses the shortest track time and the length of the experiment,
+respectively.
+
+
+First let's try to fit the dwelltime distribution to a single exponential (the simplest case). To do this we
+simply call::
+
+    dwell_1 = traces.fit_binding_times(n_components=1)
+
+This returns a `BindingDwelltimes` object which contains information about the optimized model.
+We can visually inspect the result with::
+
+    dwell_1.plot(spacing="log")
+
+.. image:: kymo_bind_dwell_1.png
+
+The `spacing` argument can be either `"log"` or `"linear"` and controls the scale of the x-axis and the spacing
+of the bin edges. You can optionally pass the number of bins to be plotted as `n_bins`. Note that the number of bins
+is purely for visualization purposes; the model is optimized directly on the unbinned dwelltimes. This is the main
+advantage of the MLE method over analyses that use a least squares fitting to binned data, where the bin widths and number
+of bins can drastically affect the optimized parameters.
+
+We can clearly see that this distribution is not fit well by a single exponential decay.
+Let's now see what a double exponential distribution looks like::
+
+    dwell_2 = traces.fit_binding_times(n_components=2)
+    dwell_2.plot(spacing="log")
+
+.. image:: kymo_bind_dwell_2.png
+
+Here we see that the double exponential fit visually looks better and the log likelihood is also higher than that
+for the single exponential fit. However, the log likelihood does not take into account model complexity, and will
+always increase for a model with more degrees of freedom. Instead, we can look at the Bayesian Information Criterion
+or Akaike Information Criterion to determine which model is better::
+
+    >>> print(dwell_1.bic, dwell_1.aic)
+    537.6232363836413  531.0366267341923
+
+    >>> print(dwell_2.bic, dwell_2.aic)
+    525.7495678897401  512.5763485908421
+
+These information criterion values weight the log likelihood with the model complexity, and as such are more useful for
+model selection. In general, the model with the lowest value is optimal. We can see that both values are lower for the double exponential model,
+indicating that it is a better fit to the data.
+
+We can next attempt to estimate confidence intervals (CI) for the parameters using a bootstrapping technique. Here, a random
+dataset with the same size as the original is sampled (with replacement) from the original dataset. The model is then
+trained on this bootstrapped data sample. This process is repeated many times, and the distribution of the resulting
+parameters can be analyzed to estimate certain statistics about the them::
+
+    dwell_2.run_bootstrap(iterations=1000)
+    dwell_2.bootstrap.inspect_component(1, alpha=0.05)
+
+.. image:: kymo_bind_bootstrap_2_1.png
+
```

Comment:
I see you added the parameter names to many of the plots, would be good to do that here too, in the plot title.